handbook/ldap/ldap.md

```diff
@@ -0,0 +1,120 @@
+
+## Enabling LDAP in SystemLink
+
+### Configuring Authentication
+
+1. Log into the server running SystemLink and open **NI Web Sever Configuration**.
+
+2. Go to the **Authentication** tab and enable **Connect to an LDAP server**.
+
+3. Enter the LDAP URL for your sever. Refer to [**LDAP URLs**](#ldap-url) for details on how to structure this URL.
+
+4. Enter the bind user. Refer to [**Bind User**](#bind-user) for details on usernames that can be used here.
+
+5. Enter the bind password.
+
+6. Click **Apply and Restart**.
+
+<figure>
+  <img src="../../img/ldap-setup.png" width="500" />
+  <figcaption>Configuration needed to enable logging into a SystemLink server using LDAP credentials.</figcaption>
+</figure>
+
+You may now login to the SystemLink server using your LDAP credentials. To access systems and data in SystemLink, configure workspace and role mappings. Refer to [**Mapping LDAP Attributes and Groups to SystemLink Workspaces and Roles**](#mapping-ldap-attributes-and-groups-to-systemlink-workspaces-and-roles) for details.
+
+<figure>
+  <img src="../../img/no-mapping.png" width="500" />
+  <figcaption>View from a user who can login but does not have any workspace and role mappings.</figcaption>
+</figure>
+
+## LDAP URL and Bind User
+
+You must provide a URL and bind user and bind password to establish an authenticated connection between your SystemLink server and LDAP server.
+
+### LDAP URL
+
+The LDAP URL follows a standard scheme. The URL determines the server to connect to, the server's port, a base search distinguished name (DN), and the attribute that determines the SystemLink username for login.
+
+!!! note "Basic LDAP URL Scheme"
+
+    **Structure:** `ldap://<server-dns>:<port>/<target-entry-dn>?<username-attribute>`
+
+    **Example:** `ldap://example.com:389/dc=example,dc=com?sAMAccountName`
+
+    In this example the server is `example.com`, the port is 389, the base search DN is `dc=example,dc=com` and the LDAP attribute used for user login is `sAMAccountName`. 
+
+!!! note "Example LDAP URLs specifying different usernames for the user Jane Doe"
+
+    **URL:** `ldap://example.com/dc=example,dc=com?sAMAccountName`
+    **Username:** jdoe
+
+    **URL:** `ldap://example.com/dc=example,dc=com?userPrincipalName`
+    **Username:** jane.doe@example.com
+
+    <figure>
+      <img src="../../img/ldap-login.png" width="500" />
+      <figcaption>Username `jdoe` is used because the `sAMAccountName` attribute is specified in the LDAP URL.</figcaption>
+    </figure>
+
+If a username attribute is not specified **NI Web Server Configuration** will automatically add `uid` as the attribute. Depending on how your LDAP directory is setup this attribute may or may not be available.
+
+### Bind User
+
+The bind user and bind password are used to authenticate with the LDAP server. Provide a `distinguishedName` or a `userPrincipalName` to specify the bind user.
```

Comment:
We may want to specify a note that this user is used when querying the LDAP server, so its password needs to either not change, or when the password is changed, you'll need to update the NI Web Server configuration with the new password.

Reply:
Added this to the intro paragraph to Bind User section.